feature-group-definitions/media-capture.yml

```
@@ -0,0 +1,42 @@
+name: Media capture
+description: TODO
```

Not sure it's a good idea exactly, but I'm following caniuse's lead here by mentioning the `getUserMedia()` entry point.

```suggestion
description: The `navigator.mediaDevices.getUserMedia()` API requests access to devices that produce streams of data, such as microphones or video cameras.
```